index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      @media screen and ( max-width: 600px ) {
+    body {
+        background-color: rgb(174, 174, 238);
+    }
+}
+    </style>
```

Comment:
I recommend adding all CSS code in a separate file (as a good practice). In your case, you already created the file, only need to move this part of your code to this file :smiley:

Reply:
✔️ Great job pointing this
- [ ] You can add the `WHY` for this change request, by adding some links for CSS/HTML best practice, what do you think @yersel500